src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -36,7 +37,7 @@ export const ChatMessage: React.FC<ChatProps> = ({
 
   return (
     <div className={containerStyles}>
-      <div className="chat-message__text">{text}</div>
+      <div className="chat-message__text">{getLinkFromText(text)}</div>
```

Comment:
See my thoughts/concerns on the potential security rammifications of exposing this functionality to users (as opposed to just admins/venue owners) without further vetting/checking our current implementation in: 

> >  But otherwise I think it should be safe and very useful UX feature very easy to implement.
> 
> We need to be careful about how this gets implemented, as there are many ways that links/URLs can be turned into a security vulnerability (eg. linking to `javascript:alert('do something malicious here')`)
> 
> ---
> 
> > possible solutions are in stackoverflow.com/questions/33235890/react-replace-links-in-a-text . Haven't spotted any react- library mentioned there already being used but didn't look too hard
> 
> We actually already have this functionality within the `AnnouncementMessage` component (used for the banners displayed at the top screen), using `getLinkFromText` from `src/utils/getLinkFromText.tsx`.
> 
> The 'risk' of that implementation is limited to venue owners/admins at the moment, so there is less scope for a malicious user to be able to take advantage of it. I would want to assess the implementation with more scrutiny before exposing that functionality to any guest that can use the chat feature.
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/issues/1406#issuecomment-841969107_

Reply:
@mike-lvov FYI: Need to check through the implementation of `getLinkFromText` from a security-mindset and ensure it's not going to allow any security vulnerabilities.

Reply:
> @mike-lvov FYI: Need to check through the implementation of getLinkFromText from a security-mindset and ensure it's not going to allow any security vulnerabilities.

As a solution, we can allow to only render links in messages from the `admins`, but I haven't thought about the implementation for that + if it will be that useful, if it's only admins who can post links

Reply:
@mike-lvov I think it'd probably be better UX if everyone was able to have their links/etc. I don't think it would take all that long to assess it from a security standpoint. Just need to dedicate the time for it. Would probably take less time to do that than to add the 'only do it for admins' feature I would expect.